native/tx.go

```diff
@@ -0,0 +1,41 @@
+package native
+
+import (
+	"context"
+	"crypto/ed25519"
+	"os"
+
+	"github.com/block-vision/sui-go-sdk/models"
+	"github.com/block-vision/sui-go-sdk/sui"
+	tmtypes "github.com/cometbft/cometbft/types"
+)
+
+func callMoveFunction(ctx context.Context, cli *sui.Client, signerAddress string, gasObj string,
+	lb *tmtypes.LightBlock) (models.TxnMetaData, error) {
+	return cli.MoveCall(ctx, models.MoveCallRequest{
+		Signer:          signerAddress,
+		PackageObjectId: os.Getenv("SMART_CONTRACT_ADDRESS"),
+		Module:          os.Getenv("SMART_CONTRACT_MODULE"),
+		Function:        os.Getenv("SMART_CONTRACT_FUNCTION"),
```

Comment:
same here, we should have our Pera client wrapper, that will have all those parameters as part of the struct, instead of having them in the logic code. Suggestion: create Pera client that will have:

```
type PeraClient struct {
    Sui sui.ISuiAPI
    Signer string
    // Sui package object ID
    Package  string
    Module string
    Function string
}

func (p PeraClient) lcUpdateCall(ctx, gasObj, lb...) (models.TxnMetaData, error) {
    ....
}

// note: private key should be part of the PeraClient struct
func (p PeraClient) execTx(ctc, tx) ...
```